packages/dialog/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@metamask/test-snap-dialog",
+  "version": "4.4.1",
+  "description": "MetaMask Dialog Test Snap",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/test-snaps.git"
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org/"
+  },
+  "files": [
+    "dist/",
+    "snap.manifest.json",
+    "images/icon.svg"
+  ],
+  "scripts": {
+    "build": "mm-snap build --eval false",
+    "build:dev": "yarn build",
+    "build:clean": "yarn clean && yarn build",
+    "clean": "rimraf 'dist/*'",
+    "start": "mm-snap watch",
+    "test": "jest --passWithNoTests",
+    "test:watch": "jest --watch",
+    "lint:eslint": "eslint . --cache --ext js,ts",
+    "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' --single-quote --ignore-path ../../.prettierignore --no-error-on-unmatched-pattern",
+    "lint": "yarn lint:eslint && yarn lint:misc --check",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
+    "lint:changelog": "yarn auto-changelog validate"
+  },
+  "dependencies": {
+    "@metamask/snaps-types": "^0.26.2"
```

Comment:
Add snap UI here

Reply:
`    "@metamask/snaps-ui": "^0.27.1",` is on line 44 already

Reply:
Ah! Then it needs to be moved to be a regular dependency instead of a dev dependency

Reply:
done, fix pushing!